ckanext/geocat/utils/xpath_utils.py

```diff
@@ -309,6 +309,9 @@ def _set_distribution_format_and_media_type(
             distribution['media_type'] = ""
     elif normed_protocol == DOWNLOAD_PROTOCOL:
         distribution['media_type'] = ""
+    elif normed_protocol in LINKED_DATA_PROTOCOL:
```

Comment:
Should this be 
```suggestion
    elif normed_protocol == LINKED_DATA_PROTOCOL:
```
?

Reply:
yep :)